README.md

```diff
@@ -4,10 +4,14 @@ Exactly what it sounds like
 ## Installation
 
 All the dependencies should be in `requirements.txt`. Hence, installation
-should be as easy as running
+should be as easy as running:
 
     pip install -r requirements.txt
 
+Install the `pre-commit` hook that checks the correct formatting:
```

Comment:
@JMatej I guess it might make sense to note that this only applies to the "development" sort of this package rather than its use :slightly_smiling_face: